server/database/connection.ts

```diff
@@ -0,0 +1,30 @@
+import { Sequelize } from 'sequelize';
+import envVars from '../config/environment';
+
+const {
+  dbDev, dbTest, dbProduction, nodeEnv,
+} = envVars;
+
```

Comment:
You can import your variables instead of import envVars

Reply:
I tried to destructure them but it didn't work!

Reply:
Solved!